src/docs/v10-vs-v11-retrospective.md

```diff
@@ -0,0 +1,643 @@
+been added in support of chromatographic separations and mass spectrometry analyses. These are subclasses of the
+new `InformationObject`, which has also become the parent of `DataObject`, which used to be a direct subclass
+of `NamedThing`.
+
+### v11.0.1
+
+![InformationObject.png](../images/InformationObject.png)
+
+### v11.0.1
+
+![Configuration.png](../images/Configuration.png)
+
+Solution-centric modelling for laboratory processes was also replaced with substance-centric modelling
+
+### Removed:
+
+- `Solution`
+- `SolutionComponent`
+
+### Added:
+
+- `PortionOfSubstance`
+- `MobilePhaseSegment`
+
+### v11.0.1
+
+![PortionOfSubstance.png](../images/PortionOfSubstance.png)
+
+### v11.0.1
+
+![MobilePhaseSegment.png](../images/MobilePhaseSegment.png)
+
+The Berkeley schema retains the `AttributeValue` hierarchy but eliminates classes like `IntegerValue`, and `UrlValue`
+whose only advantages over a values of integer or string types were never-used provenance slots.
+
+### v10.9.0
+
+![AttributeValue10.png](../images/AttributeValue10.png)
+
+### v11.0.1
+
+![AttributeValue11.png](../images/AttributeValue11.png)
+
+Classes related to metabolic reactions that a cell could carry out, based on some functionality in its genome, have been
+removed, due to potential confusion with reactions that an experimenter might carry out in order to amke a sample
+suitable for analysis.
+
+### Removed:
+
+- `Reaction`
+- `ReactionParticipant`
+
+### Added
+
+- `ChemicalConversionProcess`
+
+A very general `Instrument` class was added as a subclass of `MaterialEntity` , but without any subclasses. Knowledge
+about instruments is captured directly in NMDC's MongoDB. The instances are normalized by populating their `vendor` slot
+with a value from the `InstrumentVendorEnum` and by populating the `model` slot with a value from the
+`InstrumentModelEnum`. Mappings between these NMDC vocabularies and vocabularies from collaborators like NCBI and GOLD
+are saved in various repos like nmdc-schema and nmdc-runtime.
+
+`AnalyticalSample` was removed from the `MaterialEntity` hierarchy, as NMDC does not wish to imply that certain samples
+could be used for analyses or that other could not be used for analyses, or even that some samples a locked into an
+analytical path, such that they could not be subject to any more `MaterialProcessing`.
+
+## Refactoring of Slots
+
+The number of **slots** in the schema remains 872 after the Berkeley merger, although there are 44 slots that have been
+retired from the legacy schema and 44 that were added in the switch from v10 to v11.
+
+Many of the slot changes can be understood by looking at the Python data migration code in `nmdc_schema/migrators`
+
+A major explanation for the slot differences is the adoption of a _polymorphic_  (ie multi-shaped) _model_  for slots in
+the `Database` class, which corresponds to polymorphic collections in the v11-compliant MongoDB. In the legacy model,
```

Comment:
```suggestion
the `Database` class, which corresponds to polymorphic collections in the v11-compliant MongoDB database. In the legacy model,
```

Reply:
@eecavanna @mslarae13 I'm happy with either version. Would the two of you please make a decision?

Reply:
@eecavanna 's point makes sense to me now. So let's add the database qualifier :)

Reply:
https://github.com/microbiomedata/nmdc-schema/pull/2227/commits/3d7d2c46739b878443639c6d69ea6404496af619